boltdesignsystem/bolt
# Feature/ds 237 move no shadow demos

## Jira

[https://pegadigitalit.atlassian.net/browse/DS-237](https://pegadigitalit.atlassian.net/browse/DS-237)

## Summary

Move all no-shadow demos in pattern lab to their own directory and hide them.

## Details

Move all no-shadow demos in pattern lab to their own directory and hide them. This also involved renaming the expirements directory so that tests could be placed last permanently and hidden according to its position (last).

## How to test

- make sure all no-shadow demos are removed from the components listing
- make sure the tests listing is actually hidden
- verify that the experiments listing isnt broken from the rename
- make sure that the tests still run from their new location

## Release notes

no-shadow testing updated and experiments directory moved.
